app/bundles/EmailBundle/Entity/Email.php

```diff
@@ -219,6 +219,8 @@ class Email extends FormEntity implements VariantEntityInterface, TranslationEnt
      */
     private $clonedId;
 
+    private ?bool $continueSending = false;
```

Comment:
Doesn't need to be nullable as the default value is set to false.
```suggestion
    private bool $continueSending = false;
```